lib/autoproj/package_manifest.rb

```diff
@@ -185,6 +185,14 @@ def initialize(path, manifest)
                 super()
                 @path = path
                 @manifest = manifest
+                @env = manifest.package.ws.env
+                @condition_parser = RosConditionParser.new(@env)
```

Comment:
I'm having a conceptual issue with using the environment for this. It would force us to expose the variables in the environment.

I would very much prefer that we limit ourselves to the configuration variables.

Reply:
This is exactly the same as what is done for the ros package.xml. But i am open for suggestions for what context to use instead of manifest.package.ws.env. I will try to find where the configuration variables are stored.